src/views/product-downloads-view/components/downloads-section/downloads-section.module.css

```diff
@@ -20,6 +20,10 @@
 	/* Flex-grow to fill space when in separate row */
 	flex-grow: 1;
 	width: max-content;
+
+	@media (max-width: 402px) {
+		width: 100%;
+	}
```

Comment:
Solves this issue on smaller screens

![File](https://user-images.githubusercontent.com/20215887/189219632-ed9831ed-afe7-4a6b-a7f1-f846bfee6ccb.jpg)